vue.config-builder.js

```diff
@@ -2,7 +2,7 @@ const webpack = require('webpack')
 const exec = require('child_process').execSync
 const path = require('path')
 
-module.exports = ({ appName, appLabel, version, theme, packageAlias, root = path.resolve('.'), env = process.env.NODE_ENV }) => {
+module.exports = ({ appName, appLabel, version, theme, packageAlias, publicPath, pluginOptions = {}, alias = {}, root = path.resolve('.'), env = process.env.NODE_ENV }) => {
```

Comment:
not sure why we are making this heavier? 
why can not Cordova flavor just extend whatever config builder returns?

Reply:
I didn't think about that; will change it.